packages/web-client/src/helpers/share/role.ts

```diff
@@ -210,74 +210,74 @@ export const linkRoleInternalFolder = new LinkShareRole(
   [SharePermissions.internal]
 )
 export const linkRoleViewerFile = new LinkShareRole(
-  'viewer',
+  'view',
   false,
-  $gettext('Viewer'),
-  $gettext('viewer'),
+  $gettext('can view'),
+  $gettext('can view'),
   'eye',
   [SharePermissions.read]
 )
 export const linkRoleViewerFolder = new LinkShareRole(
-  'viewer',
+  'view',
   true,
-  $gettext('Viewer'),
-  $gettext('viewer'),
+  $gettext('can view'),
+  $gettext('can view'),
   'eye',
   [SharePermissions.read]
 )
 export const linkRoleContributorFolder = new LinkShareRole(
-  'contributor',
+  'contribute',
   true,
-  $gettext('Contributor'),
-  $gettext('contributor'),
-  'user-settings',
+  $gettext('can contribute'),
+  $gettext('can contribute'),
+  'file-add',
   [SharePermissions.read, SharePermissions.create]
 )
 export const linkRoleEditorFile = new LinkShareRole(
-  'editor',
+  'edit',
   false,
-  $gettext('Editor'),
-  $gettext('editor'),
-  'pencil',
+  $gettext('can edit'),
+  $gettext('can edit'),
+  'file-edit',
   [SharePermissions.read, SharePermissions.update]
 )
 export const linkRoleEditorFolder = new LinkShareRole(
-  'editor',
+  'edit',
   true,
-  $gettext('Editor'),
-  $gettext('editor'),
-  'pencil',
+  $gettext('can edit'),
+  $gettext('can edit'),
+  'file-edit',
   [SharePermissions.read, SharePermissions.update, SharePermissions.create, SharePermissions.delete]
 )
 export const linkRoleUploaderFolder = new LinkShareRole(
-  'uploader',
+  'upload',
   true,
-  $gettext('Uploader'),
-  $gettext('uploader'),
+  $gettext('can upload'),
+  $gettext('can upload'),
   'upload',
   [SharePermissions.create]
 )
 export const spaceRoleViewer = new SpaceShareRole(
   'viewer',
```

Comment:
@kulmann could need some help here. Changing this to "view" breaks everything. Error: TypeError: spaceRoles[permission.roles[0]] is undefined

Reply:
please only change string that are wrapped in `$gettext()`. the labels are used for referencing roles from within the code base. if you rename that it can't be found anymore -> everything explodes.

Reply:
Wouldn't that mean we get inconsistent role names and identifier names ?